lib/create-contentful-api.ts

```diff
@@ -499,6 +504,11 @@ export default function createContentfulApi<OptionType>(
         `If you want to fetch entries in all existing locales, we recommend you to use client.withAllLocales instead of the locale='*' parameter`
       )
     }
+    if ('resolveLinks' in query) {
+      console.warn(
+        'If you do not want to resolve links, we recommend you to use client.withoutLinkResolution'
```

Comment:
```suggestion
        'If you do not want to resolve links, we recommend you to use client.withoutLinkResolution instead of the resolveLinks=false parameter'
```